ADR for standard build naming (#106)
pace-neutrons/Horace: documentation/adr/0009-use-standard-naming-for-build-artifacts.md

[<-previous](0008-use-pipeline-builds.md) | next->

# 9 - Use standard naming for build artifacts

Date: 2020-Jan-19

## Status

Accepted

## Context

> The continuous builds (of `master` and the pull requests) will create many release artifacts containing the archived (`.zip` or `tag.gz`) source and binary folders.

Reviewer: ```suggestion
The continuous builds (of `master` and the pull requests) will create many release artifacts containing the archived (`.zip` or `tar.gz`) source and binary folders.
```

> These should follow a consistent naming convention so that it can be quickly determined:

Reviewer: ```suggestion
These should follow a consistent naming convention so that the following can be quickly determined:
```

- target platform
- build date

> - whether this is PR build or branch build.

Reviewer: ```suggestion
- whether this is a PR build or a branch build.
```
